windows/tools/bazel.rc

```diff
@@ -0,0 +1,6 @@
+# Windows/Envoy specific Bazel build/test options.
+
+build --experimental_shortened_obj_file_path
```

Comment:
FYI, this flag:
- is `false` by default in Bazel 0.15.2
- will be `true` by default in Bazel 0.16.0
- will be unsupported by Bazel 0.17.0